tools/depends/native/JsonSchemaBuilder/src/JsonSchemaBuilder.cpp

```diff
@@ -153,7 +153,7 @@ int main(int argc, char* argv[])
 
   out << "namespace JSONRPC" << endl;
   out << "{" << endl;
-  out << "  const char* const JSONRPC_SERVICE_ID          = \"http://xbmc.org/jsonrpc/ServiceDescription.json\";" << endl;
+  out << "  const char* const JSONRPC_SERVICE_ID          = \"http://kodi.tv/jsonrpc/ServiceDescription.json\";" << endl;
```

Comment:
remove this change please. affects RPC, needs to verify it won't break things and don't want to apply brain in this pr. you can open a separate pr for it if you cba